mcp-server-tavily/src/mcp_server_tavily/config.py

```
@@ -0,0 +1,148 @@
+            return create_facilitator_config(
+                api_key_id=self.cdp_api_key_id,
+                api_key_secret=self.cdp_api_key_secret,
+            )
+        if self.facilitator_url:
+            logger.info(f"Using public facilitator at {self.facilitator_url}")
+            return {"url": self.facilitator_url}
+        return None
+
+    @computed_field
+    @property
+    def pricing(self) -> dict[str, list[PaymentOption]]:
```

Ruff (0.14.13)

Missing docstring in public method

(D102)